build.gradle

```diff
@@ -202,92 +202,144 @@ allprojects {
+    implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.6.7'
+    implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.6.7'
+    implementation group: 'org.springframework', name: 'spring-jms', version: '5.3.23'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-activemq'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
-    implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.6.7'
-    implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: '6.0.1'
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: log4jVersion
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: log4jVersion
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: log4jVersion
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-cache'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
     implementation(group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc') {
       exclude group: 'org.apache.tomcat', module: 'tomcat-jdbc'
     }
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-security'
-    implementation group: 'org.flywaydb', name: 'flyway-core', version: '9.5.0'
-    implementation group: 'org.postgresql', name: 'postgresql', version: postgresqlVersion
-    implementation group: 'io.rest-assured', name: 'rest-assured'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web-services'
+    implementation group: 'org.springframework.security', name: 'spring-security-config'
+    implementation group: 'org.springframework.security', name: 'spring-security-web'
     /*
-    Use actual implementation of xerces 2.12.2 instead of the one bundled with Java.  The version in openJDK contains
-    a bug in XMLSchemaValidator where cvc-complex-type.4 errors are reported twice. This dependency can be removed
-    when the bug is fixed.
+      Use actual implementation of xerces 2.12.2 instead of the one bundled with Java.  The version in openJDK contains
+      a bug in XMLSchemaValidator where cvc-complex-type.4 errors are reported twice.  The bug causes some XSD validation
+      tests to fail when running under openJDK 11.0.17 and above.  This dependency can be removed when the bug is fixed.
```

Comment:
Doesn't this mean this  specification of xerces can be removed altogether i.e. we aren't using Java 11

Reply:
this needs to be used for now